py3status/modules/weather_owm.py

```diff
@@ -627,9 +638,15 @@ def _format_snow(self, wthr):
 
     def _format_wind(self, wthr):
         wind = {
-            "deg": self._jpath(wthr, OWM_WIND_DEG, ""),
-            "gust": self._jpath(wthr, OWM_WIND_GUST, ""),
-            "speed": self._jpath(wthr, OWM_WIND_SPEED, ""),
+            "deg": self._jpath(
+                wthr, OWM_WIND_DEG, self._jpath(wthr, OWN_CURRENT_WIND_DEG, "")
+            ),
+            "gust": self._jpath(
+                wthr, OWM_WIND_GUST, self._jpath(wthr, OWN_CURRENT_WIND_GUST, "")
```

Comment:
if gust/speed are not found  why replace with "" ?
Below checks will then fail:

# Speed and Gust
msec_speed = wind["speed"] if ("speed" in wind) else 0
msec_gust = wind["gust"] if ("gust" in wind) else 0

and later you'll end up with non-float value.

Reply:
good point, ty